packages/yaml/package.json

```diff
@@ -25,7 +25,7 @@
 	},
 	"dependencies": {
 		"vscode-uri": "^3.0.8",
-		"yaml-language-server": "1.14.0"
+		"yaml-language-server": "1.15.0"
```

Comment:
Should we use `^1.15.0` instead?

Reply:
I personally don’t really trust their programmatic API regarding semantic versioning, especially when a bundler is involved.

https://github.com/microsoft/vscode/issues/192144 needs to be resolved first. Once that’s sufficiently done, `yaml-language-server` exports can be properly fixed. Only then I trust `yaml-language-server` to not break programmatic use.

As maintainer of `monaco-yaml` I keep a close eye on `yaml-language-server`. I’m ok manually updating it here every once in a while. Maybe we could use `~1.15.0`.

Reply:
Got it, `~1.15.0` sound good to me.